### PR TITLE
Add ReefMod Area metrics to gpkg

### PR DESCRIPTION
**Pull request by eco-ben**
Add RM 'area' and 'habitable_proportion' to geopackage. Change GBRMPA_ID fix for COTS data matching to just a temporary change (final output matches GBRMPA provided IDs).

### CODE REVIEW

File: src/1_create_canonical.jl [hunk 1]
```diff
@@ -26,6 +31,10 @@ gbr_features = GDF.read(joinpath(DATA_DIR, "gbr_features", "Great_Barrier_Reef_F
 rename!(ac_lookup, Dict("Cscape cluster" => :cscape_cluster))
 rename!(ac_lookup, Dict(:Tempgrowth => :temp_growth, :LTMP_reef => :is_LTMP_reef))
 
+# attach correct names to rme_ids from id_list_2023_03_30.csv
```
Comment (ConnectedSystems) [inline src/1_create_canonical.jl]:
```suggestion
# Attach correct names to rme_ids from id_list_2023_03_30.csv
```

Little nitpick, use sentence case to conform with existing code style.

File: src/1_create_canonical.jl [hunk 2]
```diff
@@ -113,8 +122,14 @@ output_features .= ifelse.(ismissing.(output_features), "NA", output_features)
 string_cols = contains.(string.(typeof.(eachcol(output_features))), "String")
 output_features[!, contains.(string.(typeof.(eachcol(output_features))), "String")] .= String.(output_features[:, string_cols])
 
-# Correct incorrect GBRMPA_ID to match cots_priority data
-output_features.GBRMPA_ID = ifelse.((output_features.GBRMPA_ID .== "20198"), "20-198", output_features.GBRMPA_ID)
+# Correct incorrect RME_GBRMPA_ID to match RME v1.0.33 data
+output_features.RME_GBRMPA_ID = ifelse.((output_features.RME_GBRMPA_ID .== "20198"), "20-198", output_features.RME_GBRMPA_ID)
```
Comment (ConnectedSystems) [inline src/1_create_canonical.jl]: Sorry, why aren't we using the RME id list directly? Shouldn't `RME_GBRMPA_ID` just be a copy of the first column in the `id_list`?

File: src/2_add_cots_priority.jl [hunk 3]
```diff
@@ -39,5 +42,8 @@ rename!(RRAP_lookup, Dict(:priority=>:COTS_priority))
 RRAP_lookup.COTS_priority .= ifelse.(ismissing.(RRAP_lookup.COTS_priority), "NA", RRAP_lookup.COTS_priority)
 RRAP_lookup.COTS_priority = convert.(String, RRAP_lookup.COTS_priority)
 
+# Revert incorrect GBRMPA_ID to match data provided by GBRMPA
+RRAP_lookup.GBRMPA_ID = ifelse.(RRAP_lookup.GBRMPA_ID .== "20-198", "20198", RRAP_lookup.GBRMPA_ID)
+
```
Comment (ConnectedSystems) [inline src/2_add_cots_priority.jl]: Why do this? Why not delete Lines 8-10?

Shouldn't  `GBRMPA_ID` come from `gbr_features`? (it's `LABEL_ID` I believe)

Comment (eco-ben) [reply src/2_add_cots_priority.jl]: The COTS data contains the fixed ID ("20-198") so to join the COTS information to the dataset it needs to be temporarily fixed

Comment (ConnectedSystems) [reply src/2_add_cots_priority.jl]: Okay, sounds like a pragmatic workaround for now. We'll clean up later I guess.

**Pull request merged**